demo_with_getx_and_100ms/README.md

```diff
@@ -1,16 +1,120 @@
 # demo_with_getx_and_100ms
 
-A new Flutter project.
+A demo app using Getx and 100ms.
 
 ## Getting Started
 
-This project is a starting point for a Flutter application.
-
 A few resources to get you started if this is your first Flutter project:
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+- [100ms flutter documentation](https://www.100ms.live/docs/flutter/v2/foundation/basics)
+- [getx](https://pub.dev/packages/get)
+
+## Now we will see steps to use getx and 100ms
+
+1. Create a controller class extending GetxController.
+
+    ```dart
+      class RoomController extends GetxController{}
+    ```
+    
+3. In onInit() initialize all the necessary objects like hmssdk and creating tokens.
+
+   ```dart
+      void onInit() async {
+    hmsSdk.addUpdateListener(listener: this);
+
+    hmsSdk.build();
+    List<String?>? token = await RoomService().getToken(user: name, room: url);
+
+    if (token == null) return;
+    if (token[0] == null) return;
+
+    HMSConfig config = HMSConfig(
+      authToken: token[0]!,
+      userName: name,
+      endPoint: token[1] == "true" ? "" : "https://qa-init.100ms.live/init",
+    );
+
+    hmsSdk.join(config: config);
+    super.onInit();
+    }
+   
+   ```
+    
+4. As you get onJoin() or onPreview() callback it means join success. create a list of PeerTrackNode on onTrackUpdate() callback.
+
+
+    ```dart
+      RxList<Rx<PeerTrackNode>> peerTrackList =
+      <Rx<PeerTrackNode>>[].obs;
+      
+      @override
+  void onTrackUpdate(
+      {required HMSTrack track,
+      required HMSTrackUpdate trackUpdate,
+      required HMSPeer peer}) {
+
+
+    if (track.kind == HMSTrackKind.kHMSTrackKindVideo) {
+
+      if (trackUpdate == HMSTrackUpdate.trackRemoved) {
+        removeUserFromList(peer);
+      } else {
```

Comment:
add specific check for HMSTrackUpdate.trackAdded instead of assuming in else. There can be other types of track updates
Change to - 
if (trackUpdate == HMSTrackUpdate.trackAdded) {
...
} else if (trackUpdate == HMSTrackUpdate.trackRemoved) {
...
}